### PR TITLE
SPP-9551-clarify-abbreviations

### PR DESCRIPTION
**Pull request by joepercival**
# Description

Abbreviations of ‘IDP’ and ‘IDS’ has been used as heading and link text in the glossary which some users may not understand. This pull request will clarify the full meaning of these abbreviations.

Code has not been commented and documentation has not been written because it is not necessary for this pull request.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Technical Enhancements

# Checklist:

If any of these are not completed, please explain why in the notes.

## **Definition of Done**
**Code and merges**

- [ ] Code to be commented on where applicable 
- [ ] Documentation updated where required 
- [x] Have considered non-functional requirements such as Security, Performance, Scalability, and Fault Tolerance
- [x] I have linted the code

**Testing**

- [x] All levels of acceptance test are passing (automated, integration, manual, accessibility, etc.)
- [x] I have run the behave command to check the selenium behaviour tests pass locally
- [x] Acceptance criteria met

**Other Checks**
- [x] I have performed a self-review of my own code
- [x] I have checked for spelling errors
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes don't break anything unexpected
- [x] I have checked and updated the security.txt file where required
- [x] Up to date with the main branch

### CODE REVIEW

**Review by sanjeevz3009 — COMMENTED** [review]
Testing for reviews to show.

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dlt2qvbo16ldo.cloudfront.net/
cloudfront_id=E2KLMX0CLFUJEZ

File: content/glossary/idp.jsonnet [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  term: 'IDP',
-  related: ['IDS'],
+  term: 'Integrated Data Programme (IDP)',
+  related: ['Integrated Data Service (IDS)'],
```
[the remark this answers is not in the document]
Comment (sanjeevz3009) [reply content/glossary/idp.jsonnet:3]: Joe has clarified the issue. Resolving this comment!

File: content/glossary/idp.jsonnet [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  term: 'IDP',
-  related: ['IDS'],
+  term: 'Integrated Data Programme (IDP)',
```
[the remark this answers is not in the document]
Comment (sanjeevz3009) [reply content/glossary/idp.jsonnet]: Joe has clarified the issue. Resolving this comment!

**Review by sanjeevz3009 — DISMISSED** [review]
LGTM!

File: content/glossary/idp.jsonnet (hunk 2 above)
[the remark this answers is not in the document]
Comment (gibbardsteve) [reply content/glossary/idp.jsonnet]: Integrated Data Programme would be correct - the ticket is wrong

File: content/glossary/idp.jsonnet [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-  term: 'IDP',
-  related: ['IDS'],
+  term: 'Integrated Data Program (IDP)',
```
Comment (gibbardsteve) [inline content/glossary/idp.jsonnet:2]: It should be Programme

File: content/glossary/idp.jsonnet [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+  related: ['Integrated Data Service (IDS)'],
   meaning: |||
-    The Integrated Data Programme, the ONS team that provides and adminstrates the Integrated Data System (IDS).
+    The Integrated Data Program, the ONS team that provides and adminstrates the Integrated Data Service (IDS).
```
Comment (gibbardsteve) [inline content/glossary/idp.jsonnet:5]: It should be Programme

**Comment by ons-spp-machine-user** [comment]
DEPLOY_URL=https://dlt2qvbo16ldo.cloudfront.net/
cloudfront_id=E2KLMX0CLFUJEZ

File: content/glossary/idp.jsonnet (hunk 2 above)
[the remark this answers is not in the document]
Comment (sanjeevz3009) [reply content/glossary/idp.jsonnet:2]: Thanks, Steve, we were confused =/

**Review by gibbardsteve — APPROVED** [review]

**Pull request merged**